processing/src/main/java/org/apache/carbondata/processing/loading/model/CarbonLoadModel.java

```diff
@@ -65,8 +65,7 @@
   private String csvHeader;
   private String[] csvHeaderColumns;
   private String csvDelimiter;
-  private String complexDelimiterLevel1;
-  private String complexDelimiterLevel2;
+  private ArrayList<String> complexDelimiters = new ArrayList<>();
```

Comment:
We can do lazy initialization in the setter method. This will avoid extra memory being consumed for non complex type schemas

Reply:
done